app/assets/stylesheets/components/_important-metadata.scss

```diff
@@ -13,13 +13,10 @@
 }
 
 .app-c-important-metadata__term {
-  line-height: normal;
-
   @include media(tablet) {
     box-sizing: border-box;
     clear: left;
     float: left;
-    max-width: 11em;
     padding-right: $gutter-one-third / 2;
```

Comment:
Just spotted a vertical text alignment issue with this element on [this page](https://government-frontend-pr-558.herokuapp.com/government/statistics/prison-population-figures-2017):

![screen shot 2017-12-12 at 14 45 35](https://user-images.githubusercontent.com/861310/33890435-31bd48b4-df4b-11e7-8d46-b77b67e37a1b.png)

CSS here could be rewritten more simply to fix this issue:

.app-c-important-metadata__term {
     ...

@include media(tablet) {
     display: inline-block;
     padding-right: $gutter-one-third / 2;
}
}

.app-c-metadata__definition {
     display: inline-block;
}

Only difference is with this approach if the text of the __definition reaches the right hand side the layout defaults to the mobile one i.e. each on a separate line.

Reply:
Thanks @andysellick, @alextea and I had debated whether inline-block would be a better approach. I'll give it a try :+1:

Reply:
I've tried this approach with multiple terms and definitions and they end up on the same line instead of
```
term: definition
term: definition
```
I've tried adding a `:before` rule to display empty content as a block:
```scss
.app-c-important-metadata__term {
  content: '';
  display: block;
}
```
Which solves the listing problem, but feels a bit hacky and also leaves redundant space:
![screenshot from 2017-12-12 17-43-43](https://user-images.githubusercontent.com/93511/33899646-08262dea-df64-11e7-9b87-34ad7ff1a2e4.png)